packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -17,6 +18,15 @@ type Props = {
   shouldDeferAdapterConfiguration?: boolean;
 };
 
+type CustomLDUser = {
```

Comment:
This is out type we pass to `omitEmpty` so it can infer the types correctly from how I understand the library.